components/operators/openshift-ai/instance/components/kueue-operator/patch-datasciencecluster.yaml

```diff
@@ -0,0 +1,12 @@
+kind: DataScienceCluster
+apiVersion: datasciencecluster.opendatahub.io/v1
+metadata:
+  name: default
+spec:
+  components:
+    codeflare:
+      managementState: Managed
+    kueue:
```

Comment:
I'm wondering if it would make more sense to call this something besides `kueue-operator` since this is kind of the opposite.

I wouldn't mind updating the current `component-distributed-compute` to disable the kueue operator and including the `Red Hat build of Kueue` in the AI Accelerator by default since that will be the preferred method moving forward.

I don't think we have any sort of backwards compatibility concerns since keueu/distributed compute was not really used in any of the existing examples.

Thoughts?